dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1555,7 +1555,7 @@ Cannot.set.status.Response.already.committed=SRVE8115W: WARNING: Cannot set stat
 Cannot.set.status.Response.already.committed.explanation=After a response is committed, you cannot set the response status code.
 Cannot.set.status.Response.already.committed.useraction=Verify whether this call is necessary. If you need to set the response status code, set it before you commit the response.
 
-servlet.with.same.name.already.exists=SRVE8500W: A servlet named {0} can not be dynamically added because a servlet configuration with the same name already exists.  
+servlet.with.same.name.already.exists=SRVE8500W: A servlet named {0} cannot be dynamically added because a servlet configuration with the same name already exists.  
```

Comment:
```suggestion
servlet.with.same.name.already.exists=SRVE8500W: Servlet {0} cannot be dynamically added because a servlet configuration with the same name already exists.  
```

Reply:
Please use the "The _variable_ _noun_" format. Thus, use "The {0} servlet"